.github/workflows/code-analysis.yml

```diff
@@ -3,9 +3,9 @@ name: PHP Code Analysis
 
 on:
   pull_request:
-    branches: [ main ]
+    branches: [ applepay-feature ]
```

Comment:
You do not want to merge this to master.

Reply:
We want to merge it to the applepay-feature branch for now. When the functionality is complete we will merge the applepay-feature branch to the main branch.

Reply:
And we will revert this change when we create a pull request for the main branch.